src/EditorFeatures/DiagnosticsTestUtilities/Diagnostics/AbstractUserDiagnosticTest_GenerateTypeDialog.cs

```diff
@@ -104,7 +104,7 @@ public abstract partial class AbstractUserDiagnosticTest
 
             Assert.Equal(action.Title, FeaturesResources.Generate_new_type);
             var operations = await action.GetOperationsAsync(
-                workspace.CurrentSolution, new ProgressTracker(), CancellationToken.None);
+                workspace.CurrentSolution, CodeAnalysisProgress.None, CancellationToken.None);
```

Comment:
testing code.  ok to not have progress here. will stop commenting on this at this point.